v2ray/config.json

```
@@ -0,0 +1,5 @@
+{
+  "$schema": "../config.schema.json",
+  "ssh": false,
+  "logoUrl": "./v2ray.png"
```

We need to have the absolute url like this:
```suggestion
  "logoUrl": "https://raw.githubusercontent.com/akash-network/awesome-akash/master/v2ray/v2ray.png"
```

---

Fix it!
Thx @baktun14 !